blog/content/post/2020-08-26-coreboot-riscv.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Building coreboot for RISC-V and running it in Qemu
+abstract: 'In this article, I will show you step by step how to build coreboot for RISC-V and run it in QEMU emulator'
+cover: /covers/coreboot-logo.svg
+author: wojciech.niewiadomski
+layout: post
+published: true
+date: 2020-08-26
+archives: "2020"
+
+tags:
+  - Firmware
+  - coreboot
+  - RISC-V
+categories:
+  - Firmware
+  - Miscellaneous
+
+---
+
+Your post content
+
+# Building coreboot for RISC-V and running it in Qemu
+
+#### 1. Install required tools and libraries
```

Comment:
This should be a blog post, NOT a guide/procedure how to build. Some introduction is highly recommended that will briefly explain what this blog post will be about, what is RISC-V and why is this architecture so exciting.